test/uk/gov/hmrc/residencenilratebandcalculator/views/ResultsViewSpec.scala

```diff
@@ -91,6 +91,7 @@ class ResultsViewSpec extends HtmlSpec {
       }
 
       "contain headers for your answers" in {
+        pending
```

Comment:
Its pending awaiting the story that gets rid of the results page - and I suppose getting rid of the tests is properly part of that story, so I will add the comment.